### ISSUE TITLE
Allow for inclusion within a theme via Composer 

### ISSUE THREAD

**Issue by neilpie** [pie/pie-plugin-manager#5]
Allowing users to install this as a standalone plugin is great, but more technical users might prefer to include via Composer

**Comment by brad1066** [comment]
[Neil](@neilpie), I've looked into this, and implemented it, but have noticed that it does not allow for the plugin updater to function. Would this be something that I need to look into?

**Comment by neilpie** [comment]
@brad1066 no, if installed via a theme as a composer dependency then a developer would be pulling in the latest versions via composer and controlling it that way. This only needs to hook into updates if installed as a plugin

**Comment by brad1066** [comment]
Okey dokey. Then it should be okay to review [here](https://github.com/pie/theme-plugin-manager/commit/4af65e01c52ca7bc09620b31bacfae0d97153ed2)
